src/payment/strategies/opy/opy-script-loader.ts

```diff
@@ -0,0 +1,31 @@
+import { ScriptLoader } from '@bigcommerce/script-loader';
+
+import { PaymentMethodClientUnavailableError } from '../../errors';
+
+import { OpyHostWindow, OpyWidgets } from './opy-library';
+
+const AU_REGION_URL = 'https://widgets.openpay.com.au/lib/openpay-widgets.min.js';
+
+const SOURCES = {
+    AU: AU_REGION_URL,
+    NZ: AU_REGION_URL,
```

Comment:
Do NZ and UK have the same widget as AU?

Reply:
The Openpay team has not specified it yet but tentatively yes as AU and UK share the same script. https://widgets.openpay.com.au/quick-start

Reply:
I'll remove it until they clarify how to handle it. `NZ` is under experiment anyway.